.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: node_js
+
+sudo: false
+
+node_js:
+- 10
+- 12
+
+before_install:
+- echo -e "machine github.ibm.com\n  login $GITHUB_OAUTH_TOKEN" > ~/.netrc
+# -  'openssl aes-256-cbc -K $my_key -iv $my_iv -in myservice.env.enc -out myservice.env -d || true'
+
+script:
+- npm run build
+- npm run test-travis || travis_terminate 1
+- npm run lint
+- npm run check-packages
+# - sh scripts/typedoc/generate_typedoc.sh currently in progress
+
+
+
+after_success:
+- npm run report-coverage
```

Comment:
Is it informational or does it fail build in the event coverage is not complete?

Reply:
informational, as I don't believe we have any code-coverage constraints defined.

Reply:
If there are any code-coverage constraints, they would be applied with the `codecov` GitHub app and that would happen outside of Travis. This is purely informational